src/ray/raylet/scheduling/scheduling_policy.cc

```diff
@@ -14,41 +14,76 @@
 
 #include "ray/raylet/scheduling/scheduling_policy.h"
 
+#include <functional>
+
 namespace ray {
 
 namespace raylet_scheduling_policy {
+namespace {
+bool IsCPUOnlyRequest(const ResourceRequest &resource_request) {
```

Comment:
```suggestion
bool IsNonGPUTask(const ResourceRequest &resource_request) {
```

ISCpuOnly is misleading, since the task could be requesting other types of resources (e.g., memory).

Reply:
Actually could avoid double negative later by inverting this to "IsGPUTask"

Reply:
done